tools/image-tool/src/crane.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::{Path, PathBuf};
```

Comment:
Module-level docstring comment referencing `crane`'s documentation/home and explaining why we support it here is probably worth it.